ingest-app/project.clj

```diff
@@ -1,14 +1,22 @@
 (defproject nasa-cmr/cmr-ingest-app "0.1.0-SNAPSHOT"
   :description "Ingest is an external facing CMR service facilitating providers to create and  update their concepts in CMR. Internally it delegates concept persistence operations to metadata db and indexer micro services."
   :url "https://github.com/nasa/Common-Metadata-Repository/tree/master/ingest-app"
-  :exclusions [[commons-codec/commons-codec]
+  :exclusions [[cheshire]
+               [clj-time]
+               [com.fasterxml.jackson.core/jackson-core]
+               [commons-codec/commons-codec]
                [commons-io]
                [instaparse]
                [org.apache.httpcomponents/httpclient]
                [org.apache.httpcomponents/httpcore]
                [org.slf4j/slf4j-api]
                [ring/ring-codec]]
-  :dependencies [[camel-snake-kebab "0.4.2"]
+  :dependencies [[cheshire "5.10.0"]
```

Comment:
What are these dependency changes for?

Reply:
the CMR wouldn't compile so I was trying to fix it. It turns out that a new image container for leiningen was deployed yesterday and it broke our builds. So I will be reverting these changes back out.